public/apps/configuration/panels/role-edit/index-permission-panel.tsx

```diff
@@ -332,159 +290,26 @@ function generateIndexPermissionPanels(
-          field-level security, and tenants. Once you've created the role, you can map users to
-          these roles so that users gain those permissions.{' '}
-          <EuiLink external href="/">
-            Learn More
-          </EuiLink>
-        </EuiText>
-      </EuiPageHeader>
-      <PanelWithHeader headerText="Name">
-        <EuiForm>
-          <FormRow
-            headerText="Name"
-            headerSubText="Specify a descriptive and unique role name. You cannot edit the name onece the role is created."
-            helpText="The Role name must contain from m to n characters. Valid characters are 
-            lowercase a-z, 0-9 and (-) hyphen."
-          >
-            <EuiFieldText
-              value={roleName}
-              onChange={e => {
-                setRoleName(e.target.value);
-              }}
-              disabled={props.action == 'edit'}
-            />
-          </FormRow>
-        </EuiForm>
-      </PanelWithHeader>
-      <EuiSpacer size="m" />
-      <PanelWithHeader
-        headerText="Cluster Permissions"
-        headerSubText="Specify how users in this role can access the cluster. By default, no cluster permission is granted."
-        helpLink="/"
-      >
-        <EuiForm>
-          <FormRow
-            headerText="Cluster Permissions"
-            headerSubText="Specify permissions using either action groups or single permissions. An action group is a list of single permissions.
-            You can often achieve your desired security posture using some combination of the default permission groups. You can
-            also create your own reusable permission groups."
-          >
-            <EuiFlexGroup>
-              <EuiFlexItem style={{ maxWidth: '400px' }}>
-                <EuiComboBox
-                  options={clusterWisePermissionOptions}
-                  selectedOptions={roleClusterPermission}
-                  onChange={setRoleClusterPermission}
-                />
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiButton>Browse and select</EuiButton>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiButton iconType="popout" iconSide="right">
-                  Create Action Groups
-                </EuiButton>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </FormRow>
-        </EuiForm>
-      </PanelWithHeader>
-      <EuiSpacer size="m" />
-      <PanelWithHeader
-        headerText="Index Permissions"
-        headerSubText="Index permissions allow you to specify how users in this role can access the specific indices. By default, no index permission is granted."
-        helpLink="/"
+    <PanelWithHeader
+      headerText="Index Permissions"
+      headerSubText="Index permissions allow you to specify how users in this role can access the specific indices. By default, no index permission is granted."
```

Comment:
Looks like UI mockup has a typo. It says no cluster permission is granted.

Reply:
The descriptive text will be reviewed by tech writer and may be revised